### PR TITLE
F#186 user watchlist

### LINKED ISSUE #186: As a user, I want to have a watchlist to keep track of movies I'd like to watch (and/or have already watched)

### PR DESCRIPTION
**Pull request by hanshank**
Closes #186 

- Added a watchlist section in the user dashboard
- Not fully styled yet
- Watchlist movies are displayed, can be removed/deleted via button and are responsive
- Watchlist is now the default front-page for the user dashboard rather than the preferences page

### CODE REVIEW

File: src/components/AccountNav.js [hunk 1]
```diff
@@ -5,10 +5,16 @@ const AccountNav = () => (
...
       </li>
+      <li>
+        <NavLink to="/account/preferences">Preferences</NavLink>
+      </li>
+      {/* 
+      TO DO: Implement account settings page, where user can change password etc. 
+      */}
       {/* <li>
-        <NavLink to="/account/watchlist">Watchlist</NavLink>
+        <NavLink to="/account/settings">Account settings</NavLink>
```
Comment (glenflorendo) [inline src/components/AccountNav.js]: - [ ] Capitalize each word

File: src/components/AccountSettings.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+/* eslint-disable react/prefer-stateless-function */
```
Comment (glenflorendo) [inline src/components/AccountSettings.js]: Don't ignore this.

- [ ] Convert to stateless functional component.

File: src/components/AccountSettings.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+  render () {
+    return (
+      <div className='account-page'>
+        <h1 className='account-title'>Account Settings</h1>
+        <form className='account-settings-form'>
+          <label>Email</label>
+          <input type='text' name='email' placeholder='Type new email address'/>
+          <label>New Password</label>
+          <input name='newPassword' type='text' placeholder='Type new password' />
```
Comment (glenflorendo) [inline src/components/AccountSettings.js]: For security, you should verify the user by asking their current password.

**Comment by glenflorendo** [comment]
- [ ] Pull updates from staging

**Comment by hanshank** [comment]
Thanks Glen! Implemented the suggested changes and just commented out the settings stuff for now. I still need to make that work on the back-end.

**Comment by glenflorendo** [comment]
@hanshank Resolve your merge conflicts.

**Pull request merged**